tuner/tuner/libtuner.py

```diff
@@ -236,20 +237,59 @@ class ParsedDisptachBenchmarkResult:
 class IREEBenchmarkResult:
     # Default format follows output of iree-benchmark-module
     candidate_id: int
-    result_str: str
+    result_json: list
 
     def get_mean_time(self) -> Optional[float]:
-        if not self.result_str:
-            return None
-        pattern = r"process_time/real_time_mean\s+([\d.]+)\s\w{2}"
-        match = re.search(pattern, self.result_str)
-        if not match:
-            return None
-        try:
-            return float(match.group(1))
-        except ValueError:
+        """Compute the mean time for all of the benchmarks - The result will be in ms"""
+        if not self.result_json:
             return None
 
+        total_time = 0.0
+        count = 0
+
+        for benchmark in self.result_json:
+            real_time = benchmark.get("real_time")
+            time_unit = benchmark.get("time_unit", "ms")
```

Comment:
What does the second parameter do here?

Reply:
I found it defined like this in the documentation: `get(key, default=None).` I assumed it sets a default value in case of a missing field. If needed, I can remove it.

Reply:
The value should always be there, we don't need defaults